plugins/generator-1.19.4/forge-1.19.4/procedures/team_get_entity_team.java.ftl

```diff
@@ -1 +1,2 @@
-(${input$entity}.level.getScoreboard().getPlayersTeam(${input$entity}.getStringUUID()) != null ? ${input$entity}.level.getScoreboard().getPlayersTeam(${input$entity}.getStringUUID()).getName() : "")
\ No newline at end of file
+(${input$entity} instanceof LivingEntity _teamEnt${cbi} && _teamEnt${cbi}.level.getScoreboard().getPlayersTeam(_teamEnt${cbi}.getStringUUID()) != null ?

```

Comment:
By adding a check for living entities, you remove the possibility of adding non living entities to a team while it is supported by the game (e.g. armor stands)

Reply:
I see

Reply:
Seems ArmorStands are living entities too:

![image](https://github.com/MCreator/MCreator/assets/16374228/4d206dac-7913-49d8-9c40-f30d9bfc46a0)